kolibri2zim/scraper.py

```diff
@@ -764,15 +767,21 @@ def run(self):
             filename=self.output_dir.joinpath(self.fname),
             main_path=self.root_id,
             ignore_duplicates=True,
-            language="eng",
-            title=self.title,
-            description=self.description,
-            creator=self.author,
-            publisher=self.publisher,
-            name=self.name,
-            tags=";".join(self.tags),
-            date=datetime.date.today().strftime("%Y-%d-%m"),
-        ).start()
+        )
+        with open(self.favicon_fpath, "rb") as fh:
+            favicon_bytes = fh.read()
+        self.creator.config_metadata(
+            Name=self.name,
+            Language="eng",
+            Title=self.title,
+            Description=self.description,
+            Creator=self.author,
+            Publisher=self.publisher,
+            Date=datetime.date.today().strftime("%Y-%d-%m"),
+            Illustration_48x48_at_1= favicon_bytes,
```

Comment:
If you don't use those bytes…
```suggestion
            Illustration_48x48_at_1= self.favicon_fpath.read_bytes(),
```